crates/holochain/tests/sharded_gossip/mod.rs

```diff
@@ -201,6 +201,51 @@ async fn test_gossip_shutdown() {
     assert_eq!(record.unwrap().action_address(), &hash);
 }
 
+/// Test that when a new conductor joins, gossip picks up existing data without needing a publish.
+#[cfg(feature = "slow_tests")]
+#[tokio::test(flavor = "multi_thread")]
+async fn test_gossip_startup() {
```

Comment:
This test passed even before the fix. It could be useful, but I think I'll just ignore it, since it does sleep.